lib/db/migrations.ts

```diff
@@ -76,5 +86,48 @@ export function migrations(dbType: DatabaseDriverName) {
         await db.schema.alterTable('upload_parts').dropColumn('e_tag').execute()
       },
     },
+    $4_cache_entry_created_at: {
+      async up(db) {
+        await db.schema.alterTable('cache_keys').addColumn('created_at', 'text').execute()
+
+        await db
+          .updateTable('cache_keys')
+          .set({
+            created_at: new Date().toISOString(),
+          })
+          .execute()
+
+        if (dbType === 'mysql')
+          await db.schema
+            .alterTable('cache_keys')
+            .modifyColumn('created_at', 'text', (c) => c.notNull())
+            .execute()
+        else if (dbType === 'postgres')
+          await db.schema
+            .alterTable('cache_keys')
+            .alterColumn('created_at', (c) => c.setNotNull())
+            .execute()
+        else {
+          // rename old table
+          await db.schema.alterTable('cache_keys').renameTo('old_cache_keys').execute()
+          // recreate table
+          await createCacheKeysTable({ db, dbType })
+
+          // add not null column
+          await db.schema
+            .alterTable('cache_keys')
+            .addColumn('created_at', 'text', (c) => c.notNull())
+            .execute()
+
+          // migrate old data
+          await db
+            .insertInto('cache_keys')
+            .expression((e) => e.selectFrom('old_cache_keys').selectAll())
+            .execute()
+
+          await db.schema.dropTable('old_cache_keys').execute()
+        }
```

Comment:
alterColumn should suffice, as it should be supported in every db. Or am I wrong?